src/core/tokenHandler.js

```diff
@@ -25,9 +25,9 @@ import tokenStoreFactory from 'core/tokenStore';
 import Promise from 'core/promise';
```

Comment:
I forgot one thing. `import Promise` should no longer be necessary now we have Babel. (Same for `tokenStore.js`)

Reply:
good catch